themes/powerline/powerline.theme

```diff
@@ -8,6 +8,7 @@ _lp_powerline_theme_activate() {
     POWERLINE_ROOT_MARKER=${POWERLINE_ROOT_MARKER:-"#"}
     POWERLINE_STASH_MARKER=${POWERLINE_STASH_MARKER:-"ST"}
     POWERLINE_PYTHON_ENV_MARKER=${POWERLINE_PYTHON_ENV_MARKER:-"(e) "}
+    POWERLINE_TERRAFORM_ENV_MARKER=${POWERLINE_TERRAFORM_ENV_MARKER:-"(tf) "}
```

Comment:
I guess I need to do better documenting this: the default Powerline theme is a clone of the Powerline project, so features not in that shouldn't be added here. The powerline_full theme should have everything added to it, so keep those additions.

Reply:
I think I even saw this in the docs, but it didn't register with me when I went through and adapted all the virtualenv options as terraform options.  Also, I wanted this for myself.  :)  But I can keep that in my own fork if I need it.